components/button.css

```diff
@@ -169,4 +169,25 @@
     background: var();
     color: var(--danger);
     background: var(--grey);
-}
\ No newline at end of file
+}
+
+/*-- Ghost Button /'''\(Ⓘ.Ⓘ)/'''\ spooky--*/
+
+.btn-ghost{
+    color: var(--primary-900);
+    filter: none;
```

Comment:
filter:none;
Doesn't have any effect.

Reply:
it removes drop shadow from button component

Reply:
I think You can also use border:none;
It also does the job.

Reply:
I tried border none, it won't work for the shadow casted by the text inside the button